ddl/reorg.go

```diff
@@ -258,12 +265,16 @@ type reorgInfo struct {
 	// PhysicalTableID is used to trace the current partition we are handling.
 	// If the table is not partitioned, PhysicalTableID would be TableID.
 	PhysicalTableID int64
+	elements        []*meta.Element
```

Comment:
Is there has any possible that the `elements` order changed and we may missing reorg someone?

Reply:
The elements in "elements" will not be modified in the `column type change` operation

Reply:
But for other use case such as drop columns, will there has other DDL query may change the order and will affect the drop columns DDL query?

Reply:
But, we `elements` is set in `DeleteReorg` or `WriteReorg`. Now I will not process half of `drop columns`, and then process other operations on another ddl worker